.github/workflows/tests.yml

```diff
@@ -8,6 +8,11 @@ jobs:
   tests:
     name: Run pytest
     runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        dbt-version: ["1.6.0", "1.7.0"]
```

Comment:
We require that [this test](https://github.com/z3z1ma/dbt-osmosis/blob/a1c21093c06713fb5f5486a3e3257c7ac5153857/tests/test_column_level_knowledge_propagator.py#L7) be dbt 1.6 or higher, so the version of dbt to be run by CI is 1.6 or higher. I think that continuing to support too wide a range of multiple versions will lead to higher maintenance costs in the future for dbt-osmosis, so I personally think it is better to clearly state that the supported versions are 1.6 or higher.

The `Support level and end date` for dbt 1.5 and below is `End of Life`.

If this policy is acceptable, I will make a separate pull request to remove the dbt-osmosis code for dbt 1.5 and lower versions.

- https://docs.getdbt.com/docs/dbt-versions/core

Reply:
I'm okay with dropping support of 1.5 and lower. Its been long enough now that I would expect dbt 1.6 being a reasonable floor. Besides, people can use an older version of dbt-osmosis on pypi and dependency resolvers will handle that for them if they have a lower pinned dbt version.

Reply:
@z3z1ma Thank you very much. Now, once this pull request is merged, I will create a pull request to drop support for dbt-core 1.5 and below.